README.md

```diff
@@ -91,6 +96,9 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+If you are encountering `undefined method ‘attribute’ for ActiveModel::Serializer class` error, you might be using version `0.9.x`.
+Please see [How to Alias Attribute in 0.9.x](#/tree/0-9-stable#alias-attribute)
```

Comment:
Agree, I see the pain in here @anduong, and I know you spent some time looking up for this. 
IMO the 0.9.x link on the README solve most of the problems, should be, from now on, the first place ppl using older version should check I case of any problems. 
Taking into account that we don't have an actual troubleshooting section right now I'd vote for not having this and try to keep information contained at their versions just for sake of organization, sorry :sweat_smile: 
What we should do is to check if other ppl show up with the same doubt and then review this decision.